src/List.test.jsx

```diff
@@ -0,0 +1,51 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import List from './List';
+
+import { tasks } from '../fixtures/tasks';
+
+describe('List Component', () => {
+  const onClickDelete = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const renderComponent = (_tasks) => render((
+    <List tasks={_tasks} onClickDelete={onClickDelete} />
+  ));
```

Comment:
이미 `tasks`를 사용하고 있어서, 이름이 겹쳐서 이렇게 작성해주신 것 같아요. `_tasks`는 변수가 살아있는 기간이 매우 짧기 때문에 더 간결한 이름을 사용해도 괜찮습니다.
```suggestion
  const renderComponent = (value) => render((
    <List tasks={value} onClickDelete={onClickDelete} />
  ));
```

심지어 이렇게 해도 이해하는데 문제는 없죠